tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -33,4 +40,55 @@ describe(__filename, () => {
     expect(carousel).toHaveLength(1);
     expect(carousel).toHaveProp('random', true);
   });
+
+  it('calls tracking on home page view', () => {
```

Comment:
not sure we want this because we want to track user engagement. This will only show a 50-50 result (given that we should show the small hero UI to 50% of the users)